crates/re_sdk/src/recording_stream.rs

```diff
@@ -410,8 +417,14 @@ impl RecordingStreamInner {
             cmds_tx,
             batcher,
             batcher_to_sink_handle: Some(batcher_to_sink_handle),
+            pid_at_creation: std::process::id(),
         })
     }
+
+    #[inline]
+    pub fn has_forked(&self) -> bool {
```

Comment:
its good to clarify that we only care about the child process

```suggestion
    pub fn is_forked_child(&self) -> bool {
```